sycamore/llms/prompts/default_prompts.py

```diff
@@ -73,3 +73,53 @@
     =========
     {{gen "summary"}}
     """
+
+SCHEMA_ZERO_SHOT_GUIDANCE_PROMPT_CHAT = """
```

Comment:
Does guidance help you here? We've been going back and forth on whether it's worth it. Curious what you think.

Reply:
I'm not using any guidance-specific features at the moment.